test/util/all_close_f.cpp

```diff
@@ -299,20 +286,31 @@ ::testing::AssertionResult test::all_close_f(const vector<float>& a,
         median_distance = median_sum / 2;
     }
 
-    ar_fail << "passing criteria: " << (mantissa_bits - tolerance_bits) << " mantissa bits ("
-            << mantissa_bits << " mantissa bits w/ " << tolerance_bits << " tolerance bits)\n";
-    ar_fail << std::setprecision(std::numeric_limits<long double>::digits10 + 1)
-            << "tightest match:   " << matching_mantissa_bits(min_distance) << " mantissa bits ("
-            << a[min_distance_index] << " vs " << b[min_distance_index] << " at ["
-            << min_distance_index << "])\n";
-    ar_fail << std::setprecision(std::numeric_limits<long double>::digits10 + 1)
-            << "loosest match:    " << matching_mantissa_bits(max_distance) << " mantissa bits ("
-            << a[max_distance_index] << " vs " << b[max_distance_index] << " at ["
-            << max_distance_index << "])\n";
-    ar_fail << "median match:     " << matching_mantissa_bits(median_distance)
-            << " mantissa bits\n";
-
-    return rc ? ::testing::AssertionSuccess() : ar_fail;
+    if (rc)
```

Comment:
Can we condition these with an env var or something? Unobtrusive is in the eye of the beholder.

Reply:
Now quiet by default.